tools/BUILD

```diff
@@ -16,3 +18,23 @@ py_binary(
         "//:nighthawk_test_server",
     ],
 )
+
+py_binary(
+    name = "format_python_tools",
+    srcs = ["format_python_tools.py"],
+    data = [
+        ":.style.yapf",
+    ],
+    deps = [
+        requirement("yapf"),
+    ],
+)
+
+py_binary(
+    name = "flake8",
```

Comment:
you can use an `entry_point` for this

Reply:
I cannot, there is no way of including the plugin `flake8-docstrings` with an `entry_point`. This is the only workaround I found.

Reply:
fwiw - we have `envoy_entry_point` to workaround this specific issue

https://github.com/bazelbuild/rules_python/issues/600